test/specs/spaniel-observer.spec.js

```diff
@@ -203,4 +203,26 @@ describe('SpanielObserver', function() {
       })
       .then(cleanUp);
   });
+
+  it('should start paused if in a background tab', function() {
```

Comment:
Can you also add a test that no impression hook is fired if the tab is then closed?